dod-wrapper/src/main/java/com/revolut/rxdata/dod_wrapper/DataObservableDelegateWrapperDispatchers.kt

```diff
@@ -0,0 +1,36 @@
+package com.revolut.rxdata.dod_wrapper
+
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.Dispatchers
+import org.jetbrains.annotations.TestOnly
+
+/*
+ * Copyright (C) 2023 Revolut
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+internal object DataObservableDelegateWrapperDispatchers {
+
+    internal var IO = Dispatchers.IO
+        private set
+    internal var Unconfined = Dispatchers.Unconfined
+        private set
+
+    @TestOnly
+    fun setDispatchers(dispatcher: CoroutineDispatcher) {
```

Comment:
this doesn't seem to be used anymore (did coroutine version upgrade removed the necessity to override dispatchers in tests?), maybe remove it and just use Dispatchers.IO where needed?

Reply:
We don't use it here in unit tests, because Turbine waits for values to be processed by "real" dispatchers.

But consumers might still need to pass custom dispatchers to achieve synchronous processing if they use another testing infrastructure (e.g. we ourselves would need it in Revolut).